db/fixtures/ae_datastore/ManageIQ/System/Process.class/miqevent.yaml

```diff
@@ -0,0 +1,12 @@
+---
+object_type: instance
+version: 1.0
+object:
+  attributes:
+    display_name: 
+    name: MiqEvent
+    inherits: 
+    description: 
+  fields:
+  - rel5:
+      value: "/System/Event/MiqEvent/Policy/${/#event_type}"
```

Comment:
@tinaafitz I think you want the relationship to be more like the `Request` instance in the Process class which uses
`/System/Request/${#request}`

Here I would expect it to be: `/System/Event/MiqEvent/Policy/${/#request}`

Then in your example you would specify the request (which your screenshot shows) but you do not have to pass duplicate data in the Attribute/Value Pairs.

Does that work better here?